core/web/router.go

```diff
@@ -263,6 +263,7 @@ func v2Routes(app chainlink.Application, r *gin.RouterGroup) {
 		cc := ConfigController{app}
 		authv2.GET("/config", cc.Show)
 		authv2.PATCH("/config", cc.Patch)
+		authv2.GET("/config/dump", cc.Dump)
```

Comment:
Would it be better to start a `/config/v2`? Seems cleaner, especially if we just `GET` that directly: 
```suggestion
		authv2.GET("/config/v2", cc.Dump)
```
I thought we might be able to reuse the existing `GET /config` with a different media type, e.g. `application/toml` instead of `application/json`, but TOML doesn't actually have a registered media type :thinking: 
Perhaps it is preferrable to return the toml string inside a JSON field for consistency anyways :shrug:

Reply:
Yes `/config/v2` seems good to me, `/config` will become an alias to it anyway in a future version